islandora_piwik.module

```diff
@@ -252,7 +254,9 @@ EOQ;
 EOQ;
         $connection = islandora_get_tuque_connection();
         $results = $connection->repository->ri->sparqlQuery($query, 1);
-        $collection_pids[] = $results[0]['collection']['value'];
+        if (count($results)) {
+          $collection_pids[] = $results[0]['collection']['value'];
+        }
       }
     }
     return $collection_pids;
```

Comment:
You are still returning collection_pids. So what happens now if there are no collection_pids (because the results ==0) ?

Reply:
Good question! I have no idea how Piwik would handle this, to be honest, nor how I'd test it.